### PR TITLE
add assert to metadata test

### PR DESCRIPTION
**Pull request by zajca**
https://keboola.slack.com/archives/C04813S29UJ/p1666763637937079

Before asking for review make sure that:

## Checklist

- [x] New client method(s) has tests
- [x] Apiary file is updated

## Release

  - [x] I gave the PR a proper label:
    * major (BC break)
    * minor (new feature)
    * patch (backwards compatible fix)
    * no release (just test changes)

### CODE REVIEW

**Review by martinjunger — APPROVED** [review]
LGTM

File: tests/File/CommonFileTest.php [hunk 1]
```diff
@@ -279,13 +279,16 @@ public function testFilesPermissions(): void
...
         $this->assertCount(1, $files);
-        $this->assertEquals($newFileId, reset($files)['id']);
+        /** @var array{id:int} $reset */
+        $reset = reset($files);
+        $this->assertEquals($newFileId, $reset['id']);
 
         // new file should be visible for master token
         $files = $this->_client->listFiles();
-        $this->assertEquals($newFileId, reset($files)['id']);
+        $this->assertEquals($newFileId, $reset['id']);
```
Comment (ujovlado) [inline tests/File/CommonFileTest.php:291]: @zajca pride mi, ze tu chyba este znovu:

```php
$reset = reset($files);
```
lebo to robi novy fetch

Comment (zajca) [reply tests/File/CommonFileTest.php:291]: Ty jo dobrý catch, máš pravdu, fixnu to.

Comment (ujovlado) [reply tests/File/CommonFileTest.php:291]: Mozeme to prihodim k tomu mojmu PR? https://github.com/keboola/storage-api-php-client/pull/910

**Pull request merged**